ddtrace/tracer/spancontext.go

```diff
@@ -269,6 +297,16 @@ func (t *trace) finishedOne(s *span) {
 		t.root.setMetric(keySamplingPriority, *t.priority)
 		t.locked = true
 	}
+	if len(t.spans) > 0 && s == t.spans[0] {
+		// first span in chunk finished, lock down the tags
+		//
+		// TODO(x): make sure this doesn't happen in vain when switching to
+		// the new wire format. We won't need to set the tags on the first span
+		// in the chunk there.
```

Comment:
Can we put a name in the `TODO`?  The `(x)` is not helpful. We should have some trail, however minimal, to follow when actually following up on TODOs.